### PR TITLE
Message web server

### PR DESCRIPTION
**Pull request by heuermh**

### CODE REVIEW

File: message_web_server/message_web_server.ino [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  do {
+    wifiStatus = WiFi.begin(wifiSsid, wifiPass);
+    delay(500);
+  } while (wifiStatus != WL_CONNECTED);
+
+  clear();  
+  wave(60);
+
+  clear();
+  matrix.println(WiFi.localIP());
```
Comment (heuermh) [inline message_web_server/message_web_server.ino:112]: @rmheuer is there a way to reset the cursor for println back to the first line?

File: message_web_server/message_web_server.ino [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    matrix.show();
+    delay(40);
+  }
+}
+
+void loop(void) {
+  // listen for incoming clients
+  WiFiClient client = server.available();
+
+  if (client) {
```
Comment (heuermh) [inline message_web_server/message_web_server.ino:148]: adapted from example here
https://github.com/khoih-prog/WiFiWebServer/blob/master/examples/WebServer/WebServer.ino

File: message_web_server/message_web_server.ino (hunk 1 above)
Comment (rmheuer) [reply message_web_server/message_web_server.ino:112]: Yes, there is matrix.setCursor(x, y)

**Review by rmheuer — APPROVED** [review]

**Comment by rmheuer** [comment]
Seems good, I know it works because I was there when it was working

**Comment by heuermh** [comment]
Thank you, @rmheuer

**Pull request merged**